tests/Makefile

```diff
@@ -0,0 +1,6 @@
+LIBS=$(shell pkg-config --libs gtk+-3.0) -lm ../build/libgoatplot.so
```

Comment:
Please just add another test program entry to the wscript in the root of this repository.
Please rename the the test to glade-line.c
```
	test_glade = bld.program(
		features = ['c', 'glib2', 'unites'],
		target = 'test-glade-line',
		source = ['tests/glade-line.c'],
		includes = ['src/'],
		export_includes = ['src/'],
		use = 'objects',
		uselib = 'M GOBJECT GLIB GTK3',
		install_path = None
	)

```